utils/formUtils.ts

```diff
@@ -151,6 +151,9 @@ export function setStepsData(splitSchema: SplitSchema, setSplitSchema: Function,
 export function validateForm(step: Step) {
   const validator = new Validator()
   const sectionErrors = validator.validate(step.state, step.schema)
+  // This is temporay validation fix - dependency questions that rely on yes or no answers in the schema
+  // will have their data persist in the form object, even if the parent question's answer has been changed
+  sectionErrors.errors.filter((e) => e.name !== 'dependencies')
```

Comment:
I see the following errors:

```
0: Object { name: "additionalProperties", property: ".endDate['date']", message: "is an invalid additional property", … }
​​1: Object { name: "enum", property: ".endDate.hasEndDate", message: "should be equal to one of the allowed values", … }
​​2: Object { name: "oneOf", property: ".endDate", message: "should match exactly one schema in oneOf", … }
```

It still does not let me click 'next'.
<img width="1143" alt="Screenshot 2022-03-14 at 16 25 13" src="https://user-images.githubusercontent.com/89583054/158216796-182cbb32-a6c1-4633-885f-5406b467b213.png">